### ISSUE TITLE
qute://history/ is blank

### ISSUE THREAD

**Issue by mrpewpew** [qutebrowser/qutebrowser#4257]
I just noticed the `:history` page is totally empty but I still get the history when I try to open a new tab/window.
I checked `~/.local/share/qutebrowser/history.sqlite` and it's not empty. 
Any idea what the problem is ?
```
qutebrowser v1.4.2
Git commit: 
Backend: QtWebEngine (Chromium 65.0.3325.230)
```

**Comment by The-Compiler** [comment]
It's a bug in Qt 5.11.2 which already has a workaround commited (but not released yet). I assume you use that Qt version - are you on Archlinux?

**Comment by The-Compiler** [comment]
Some people reported seeing this again after the Qt 5.12 upgrade - that's indeed the case, but 530befd68154440348c776a7dbf3683a7d90d9d1 fixed it, so all will be fine with v1.6.0.
